packages/palette/package.json

```diff
@@ -105,7 +105,7 @@
     "simple-progress-webpack-plugin": "1.1.2",
     "static-extend": "0.1.2",
     "storybook-chromatic": "1.3.3",
-    "styled-components": "4.3.2",
+    "styled-components": "^5.0.0",
```

Comment:
Should this constraint be `> 4.3.2` or something? There doesn't appear to be breaking changes in Palette (🤞 ), and this might make it easier to for consumers to keep up to date with Palette.

Reply:
This is a devDependency, it won't affect consumers. You need to update the  `peerDependency` field.